### ISSUE TITLE
SQL Server LIKE filter escaping

### ISSUE THREAD

**Issue by stmeyer** [FriendsOfCake/search#143]
Hey,

when using a `LIKE` filter with sql server and the search string contains `_` the value is escaped `\_`. By searching for `example_text` it is searched for `example\_text`. But in SQL Server one has to specify the escape character with the `ESCAPE` keyword (https://msdn.microsoft.com/en-us/library/ms179859.aspx):
```sql
INSERT mytbl2 VALUES ('Discount is 10-15% off'), ('Discount is .10-.15 off');  
GO  
SELECT c1   
FROM mytbl2  
WHERE c1 LIKE '%10-15!% off%' ESCAPE '!';  
```
Do you know a way to send the `ESCAPE` keyword?

Thanks

**Comment by stmeyer** [comment]
Any ideas? Maybe this function is not implemented in core. I can create issue in core if needed.

**Comment by dereuromark** [comment]
I bet we can, based on the driver, use a specific behavior here.
Are you able to make a PR?

**Comment by stmeyer** [comment]
closed because PR #149 is open
